### PR TITLE
2022-01-19-distilbert_token_classifier_typo_detector_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "en")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = DistilBertForTokenClassification.pretrained("distilbert_token_classifier_typo_detector", "en"))\
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md:50]: There are 2 parenthesis, one only needed.

File: docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+val sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "en")
+       .setInputCols(Array("document"))
+       .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()
+      .setInputCols(Array("sentence"))
+      .setOutputCol("token")
+
+val tokenClassifier = DistilBertForTokenClassification.pretrained("distilbert_token_classifier_typo_detector", "en"))\
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md:77]: Two parenthesis, only one needed. And remove also `\` in Scala is not needed as you correctly told me some time ago :)

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
There are a couple of typos in the python/scala code. Can you check?

File: docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md]: @josejuanmartinez 
it is fixed!

File: docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-01-19-distilbert_token_classifier_typo_detector_en.md]: @josejuanmartinez 
it is fixed!

**Review by josejuanmartinez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**